src/data/model/dndModel.tsx

```diff
@@ -1,23 +1,39 @@
 export const mapperClassList = [
   "class",
-  "adept",
-  "aristocrat",
+  /* "adept", */
+  /* "aristocrat", */
   "barbarian",
   "bard",
   "cleric",
-  "commoner",
+  /* "commoner", */
   "druid",
-  "expert",
+  /* "expert", */
   "fighter",
   "monk",
   "paladin",
   "ranger",
   "rogue",
   "sorcerer",
-  "warrior",
+  "warlock",
+  /* "warrior", */
   "wizard",
 ] as const;
 export type DnDClass = typeof mapperClassList[number];
 
+export const dndClass = {
+  class_levels: "",
+  hit_die: 0,
+  index: "",
+  name: "",
+  proficiencies: [],
+  proficiency_choices: [],
+  saving_throws: [],
+  starting_equipment: [],
+  starting_equipment_options: [],
+  subclasses: [],
+  url: "",
+} as const;
+export type DnDClassStats = typeof dndClass;
```

Comment:
Instead of doing like this, I think we should define an interface first, then use that interface for `dndClass`. The reason is that something like `url` can be automatically be interpreted as `string`, but `starting_equipment_options` is only interpreted as `array of any` which defeats the purpose of type annotation
Moreover, we don't use the value of `dndClass` anywhere, so just defining an interface is much clearer. Lemme know if you need help refactoring this 👍